src/test/java/org/broadinstitute/dsp/ontology/performance/scenarios/MatchingV3.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.dsp.ontology.performance.scenarios;
+
+import com.google.api.client.http.HttpStatusCodes;
+import io.gatling.javaapi.core.ScenarioBuilder;
+import org.broadinstitute.dsp.ontology.performance.Endpoints;
+import java.util.List;
+
+import static io.gatling.javaapi.core.CoreDsl.scenario;
+import static io.gatling.javaapi.http.HttpDsl.status;
+
+public class MatchingV3 implements Endpoints {
+
+  private final String v3MatchPair = """
+      {
+        "purpose": {
+          "generalUse": true
+        },
+        "consent": {
+          "generalUse": true
+        }
+      }
+      """;
```

Comment:
This is fine as is. Another way to do this is to generate a v3 object and call `toString` on it.